aws/ec2.go

```diff
@@ -110,3 +113,394 @@ func nukeAllEc2Instances(session *session.Session, instanceIds []*string) error
 	logging.Logger.Infof("[OK] %d instance(s) terminated in %s", len(instanceIds), *session.Config.Region)
 	return nil
 }
+
+func GetEc2ServiceClient(region string) ec2iface.EC2API {
+	return ec2.New(newSession(region))
+}
+
```

Comment:
I am inclined to move all the VPC stuff to a different file, but don't have super strong feelings about it.

Reply:
The reason I kept it all in `ec2.go` is because that's how AWS has it organized. All the VPC methods are in the `ec2` package. Not that we necessarily have to follow their lead, which arguably doesn't make sense to customers, but that's the reasoning.

Reply:
Ah gotcha. It's less for customers (customers won't see this code) and more for us. I find it more intuitive for them to be separate because I spend more time in the console than the AWS Go SDK, and VPC has its own console vs EC2.

Come to think of it, I wonder why the go SDK has VPC under EC2 when the console separates them out...